examples/slash_commands/handlers.go

```diff
@@ -0,0 +1,236 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+func SelectHandler(s *discordgo.Session, i *discordgo.InteractionCreate) {
```

Comment:
This is not a critical problem, but I think this should be renamed to something else, it is quite confusing, since we have select components.

Reply:
Ok, I will look into this on weekend.

Reply:
It's done. Take a look.